projects/mintswap/index.js

```diff
@@ -0,0 +1,82 @@
+const sdk = require("@defillama/sdk");
+const erc20 = require("../helper/abis/erc20.json");
+const abi = require("./abi.json");
+const { unwrapUniswapLPs } = require("../helper/unwrapLPs");
+const { transformAvaxAddress } = require("../helper/portedTokens");
+
+const FARMING_CONTRACT_ADDRESS = "0x7Bf4cA9aEC25aDAAf7278EEDbe959d81893e314F";
+
+const avaxTvl = async (timestamp, ethBlock, chainBlocks) => {
+  const balances = {};
+
+  const lengthOfPool = (
+    await sdk.api.abi.call({
+      abi: abi.poolLength,
+      target: FARMING_CONTRACT_ADDRESS,
+      chain: "avax",
+      block: chainBlocks["avax"],
+    })
+  ).output;
+
+  const lpPositions = [];
+
+  for (let index = 0; index < lengthOfPool; index++) {
+    const lpOrTokens = (
+      await sdk.api.abi.call({
+        abi: abi.poolInfo,
+        target: FARMING_CONTRACT_ADDRESS,
+        params: index,
+        chain: "avax",
+        block: chainBlocks["avax"],
+      })
+    ).output[0];
+
+    const lpOrToken_bal = (
+      await sdk.api.abi.call({
+        abi: erc20.balanceOf,
+        target: lpOrTokens,
+        params: FARMING_CONTRACT_ADDRESS,
+        chain: "avax",
+        block: chainBlocks["avax"],
+      })
+    ).output;
+
+    if (
+      index == 12 ||

```

Comment:
this is bound to break as more tokens are added, what do you think about using addFundsInMasterchef?